modules/network/nsgs.tf

```diff
@@ -149,19 +149,6 @@ resource "oci_core_network_security_group_security_rule" "workers_egress" {
   count = length(local.workers_egress)
 }
 
-resource "oci_core_network_security_group_security_rule" "workers_egress_flannel" {

```

Comment:
Why is this being removed?

Reply:
It's also needed for NPN - I moved it to the shared locals.workers_egress since the cni_type conditional isn't needed.

Reply:
I was going by the docs. Let's try getting the rules more explicitly documented